packages/model/addon/-private/attr.js

```diff
@@ -158,7 +158,7 @@ function attr(type, options) {
           if (errors.get(key)) {
             errors.remove(key);
           }
-          this._markInvalidRequestAsClean();
+          this.___recordState.cleanErrorRequests();
```

Comment:
This used to clear out the invalid requests, but now clears out the error requests?